packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -168,6 +173,11 @@ export interface ISemanticColors extends ISemanticTextColors {
    */
   inputBackgroundChecked: string;
 
+  /**
+   * The alternate background color of a checked control, e.g. slider background, spinner background.
```

Comment:
"alternate" has no semantic meaning, it just means "another option", so we should try avoid using that word as much as possible.
Could this be like inputBackgroundCheckedGhost, or Sub instead of ghost (as in sub-text)? For example, if you hovered over an unchecked radio button and you saw the ghost of the dot.

Reply:
What do you mean by ghost? What is this alternate actually for (as in why is there an alternate)?

Reply:
You're right, "alternate" was a poor choice of wording. The background color is used for this, in addition to spinner and calendar's selected date background (that will be updated in a separate PR)